docs/sql-ref-syntax-ddl-alter-view.md

```diff
@@ -19,4 +19,217 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
```

Comment:
Might add a blank line after every markdown section, for clarity

Reply:
ok, I will add a blank line for the same level of section as `### Description`